install.sh

```diff
@@ -169,6 +169,13 @@ echo
 
 install_snowboy
 
+echo "Installing Mounting Source"
+cd $DIR_PATH
+git clone https://github.com/MrVallentin/mount.py
```

Comment:
After changing to _udisks_, why still install this garbage?

Reply:
Read carefully my comment:

> After changing to udisks

Use _udisks_, not that _mount.py_ rubbish.

Reply:
@hongquan , there is no command in `udisks` to get mount points

Reply:
I have read the udisks docs , very carefully
> there is no command in udisks to get mount points